browser/resources/settings/brave_rewards_page/brave_rewards_page.html

```diff
@@ -0,0 +1,35 @@
+<link rel="import" href="chrome://resources/html/polymer.html">
+
+<link rel="import" href="chrome://resources/html/i18n_behavior.html">
+<link rel="import" href="../brave_clear_rewards_data_dialog/brave_clear_rewards_data_dialog.html">
+<link rel="import" href="../route.html">
+<link rel="import" href="../settings_page/settings_subpage.html">
+<link rel="import" href="../settings_shared_css.html">
+<link rel="import" href="../settings_vars_css.html">
+
+<dom-module id="settings-brave-rewards-page">
```

Comment:
We should probably name this module something more specific for what it is: The Brave Rewards additions to the Privacy & Security Page, so something like `settings-brave-rewards-privacy-security`. We can leave off 'Page' since it's not accurate in this instance IMO.

Reply:
Renamed. Just FYI, some of the Cr pages that aren't 'pages' but just a bunch of sections are suffixed with `_page`